api/src/middleware/auth.js

```diff
@@ -10,7 +10,6 @@ module.exports = (req, res, next) => {
     } else {
         // JWT auiwgwjhfgioajoij
```

Comment:
I think this middleware works wrong, you should not let the requests without `authorization` header pass through, because the user will be able to use all the routes without it, for example through postman. So if the user has now auth header, you should throw unauthorized error. You are using this middelware for all requests, but I would suggest to use it for all routes, except login and registration.